### ISSUE TITLE
hey so my phone audio not working and it keeps crashing when i try to open it 

### ISSUE THREAD

**Issue by HHgshs** [Genymobile/scrcpy#3856]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [e.g. Debian, Windows, macOS...]
 - scrcpy version: [e.g. 1.12.1]
 - installation method: [e.g. manual build, apt, snap, brew, Windows release...]
 - device model:
 - Android version: [e.g. 10]

**Describe the bug**

On errors, please provide the output of the console (and `adb logcat` if relevant).

```
Please paste terminal output in a code block.
```

Please do not post screenshots of your terminal, just post the content as text instead.

**Comment by HHgshs** [comment]
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\aran\Downloads\scrapy\scrcpy-win64-v2.0\scrcpy-win64-v2.0>scrcpy --list
-encoders
scrcpy 2.0 <https://github.com/Genymobile/scrcpy>
C:\Users\aran\Downloads\scrapy\scrcpy-...ped. 27.0 MB/s (52867 bytes in 0.002s)
[server] INFO: List of video encoders:
    --video-codec=h264 --video-encoder='c2.qti.avc.encoder'
    --video-codec=h264 --video-encoder='OMX.qcom.video.encoder.avc'
    --video-codec=h264 --video-encoder='c2.android.avc.encoder'
    --video-codec=h264 --video-encoder='OMX.google.h264.encoder'
    --video-codec=h265 --video-encoder='c2.qti.hevc.encoder'
    --video-codec=h265 --video-encoder='OMX.qcom.video.encoder.hevc'
    --video-codec=h265 --video-encoder='c2.qti.hevc.encoder.cq'
    --video-codec=h265 --video-encoder='OMX.qcom.video.encoder.hevc.cq'
    --video-codec=h265 --video-encoder='c2.android.hevc.encoder'
[server] INFO: List of audio encoders:
    --audio-codec=opus --audio-encoder='c2.android.opus.encoder'
    --audio-codec=aac --audio-encoder='c2.android.aac.encoder'
    --audio-codec=aac --audio-encoder='OMX.google.aac.encoder'

C:\Users\aran\Downloads\scrapy\scrcpy-win64-v2.0\scrcpy-win64-v2.0>scrcpy --audi
o-codec=aac
scrcpy 2.0 <https://github.com/Genymobile/scrcpy>
C:\Users\aran\Downloads\scrapy\scrcpy-...ped. 30.4 MB/s (52867 bytes in 0.002s)
[server] INFO: Device: Xiaomi 21121210G (Android 13)
INFO: Renderer: direct3d
INFO: Initial texture: 1080x2400
ERROR: Could not open audio device: WASAPI can't initialize audio client: CoInit
ialize has not been called.
ERROR: Demuxer error
WARN: Killing the server...

**Comment by rom1v** [comment]
There was a WASAPI issue in SDL: https://github.com/libsdl-org/SDL/issues/7478 (fixed for the future SDL released).

Meanwhile, you can use another driver:

```cmd
set SDL_AUDIODRIVER=directsound
scrcpy
```

Then, you will probably face another problem with directsound: it does not support low latency audio (https://github.com/Genymobile/scrcpy/issues/3793).

So you need this version: https://github.com/Genymobile/scrcpy/issues/3793#issuecomment-1465713098
and execute:

```cmd
set SDL_AUDIODRIVER=directsound
scrcpy --audio-output-buffer=10
```

**Comment by HHgshs** [comment]
after trying it out stilll didnt work for me

**Comment by rom1v** [comment]
> the second i dont know why but it keeps giving me unknown option

You need the build from this comment: https://github.com/Genymobile/scrcpy/issues/3793#issuecomment-1465713098

> after trying it out stilll didnt work for me

What did you execute (with which binary) and what is the console output?

**Comment by HHgshs** [comment]
C:\Users\aran\Downloads\scrcpy-win64-v2.0>SDL_AUDIODRIVER=directsound
'SDL_AUDIODRIVER' is not recognized as an internal or external command,
operable program or batch file.

**Comment by rom1v** [comment]
You forgot `set ` before `SDL_AUDIODRIVER=directsound`. Please re-read the previous comments carefully.

**Comment by bhuvanesh15** [comment]
> There was a WASAPI issue in SDL: [libsdl-org/SDL#7478](https://github.com/libsdl-org/SDL/issues/7478) (fixed for the future SDL released), see #3825.
> 
> Meanwhile, you can use another driver:
> 
> ```batchfile
> set SDL_AUDIODRIVER=directsound
> scrcpy
> ```
> 
> Then, you will probably face another problem with directsound: it does not support low latency audio (#3793).
> 
> So you need this version: [#3793 (comment)](https://github.com/Genymobile/scrcpy/issues/3793#issuecomment-1465713098) and execute:
> 
> ```batchfile
> set SDL_AUDIODRIVER=directsound
> scrcpy --audio-output-buffer=10
> ```
> 
> (or just disable audio: `scrcpy --no-audio`)

this works for me
